runtime/oti/VMHelpers.hpp

```diff
@@ -343,11 +343,13 @@ class VM_VMHelpers
 	methodHandleCompiledEntryPoint(J9JavaVM *vm, J9VMThread *currentThread, j9object_t methodHandle)
 	{
 		void *compiledEntryPoint = NULL;
+#if defined(J9VM_OPT_METHOD_HANDLE)
```

Comment:
This whole function needs to be removed when not using out method handle.

Reply:
I'll soften this to "may", though since these are all inline functions, returning 0 should generate the same code.

Reply:
Defer to later cleanup.